test/Client-test.js

```diff
@@ -10,34 +11,94 @@ const should = chai.should()
 
 const expect = chai.expect;
 
-describe.only('Client', function() {
+describe('Client', function() {
   let user;
-  global.window = {}
-
-  // chai.spy.on(window, 'fetch', () => {
-  //   return new Promise()
-  // });
-
+  let destinationsData;
+  let tripsData;
 
 
 
   beforeEach(function() {
-    user = new Client()
+    destinationsData =
```

Comment:
Your mock data is repeated across all test files. There might be a way to consolidate.